tests/e2e/transaction/helper.ts

```diff
@@ -39,11 +39,36 @@ export async function publishPackage(
 }
 
 export async function fundAccounts(aptos: Aptos, accounts: Array<Account>) {
-  // eslint-disable-next-line no-restricted-syntax
-  for (const account of accounts) {
-    // eslint-disable-next-line no-await-in-loop
-    await aptos.fundAccount({ accountAddress: account.accountAddress.toString(), amount: FUND_AMOUNT });
-  }
+  // Fund first account
+  const firstAccount = accounts[0];
+  await aptos.fundAccount({ accountAddress: firstAccount.accountAddress.toString(), amount: FUND_AMOUNT });
+  // Fund again because these txns can be expensive
+  await aptos.fundAccount({ accountAddress: firstAccount.accountAddress.toString(), amount: FUND_AMOUNT });
+  const addressesRemaining = accounts.slice(1).map((account) => account.accountAddress);
+  const amountToSend = Math.floor((FUND_AMOUNT * 2) / accounts.length);
+  // Send APT to the rest
+  const transaction = await aptos.generateTransaction({
+    sender: firstAccount.accountAddress.toString(),
+    data: {
+      function: `0x${1}::aptos_account::batch_transfer`,
```

Comment:
Lol why like this?  If this is doing a pattern match that makes you do something like this, we need to fix it and remove the pattern match

Reply:
Lol I have no idea why I did this- it's fine with just the string, I updated it